src/NetworkAlias.php

```diff
@@ -389,10 +389,10 @@ public static function showForNetworkName(NetworkName $item, $withtemplate = 0)
     /**
      * Show the aliases contained by the alias
      *
-     * @param CommonGLPI $item          the FQDN owning the aliases
+     * @param FQDN       $item          the FQDN owning the aliases
      * @param integer    $withtemplate  withtemplate param
      **/
-    public static function showForFQDN(CommonGLPI $item, $withtemplate)
+    public static function showForFQDN(FQDN $item, $withtemplate)
```

Comment:
While this is correct, I do not know if such a change should not be considered as a BC break :/

Reply:
It fixes these errors, so can we use CommonDBTM instead? Which is probably less risky, but seems less relevant to me.
_The function is only called once in the core_
```
src/NetworkAlias.php:402:Call to an undefined method CommonGLPI::check().
src/NetworkAlias.php:402:Call to an undefined method CommonGLPI::getID().
src/NetworkAlias.php:415:Call to an undefined method CommonGLPI::getID().
src/NetworkAlias.php:441:Access to an undefined property CommonGLPI::$fields.
src/NetworkAlias.php:461:Call to an undefined method CommonGLPI::getID().
```

Reply:
I guess (for once) @var FQDN would be enough here.
FYI on main the type has already been fixed.

Reply:
Well.. Problem is potential call from plugins; not from core. But calling with something else than a `FQDN` object will fail so... I guess we can change finally.
Let's see for other reviewers pov ;)

Reply:
Same here, method signature should stay as it was. Just changing phpdoc should be enough as well at this point.